### ISSUE TITLE
var::Data performance test shouldn't use rand() (stress test should use rand)

### ISSUE THREAD

**Issue by tyler-gilbert** [StratifyLabs/sostest#52]

**Comment by tyler-gilbert** [comment]
@shomagan Did you see this issue?

I haven't looked closely at how the stress and performance tests are done but the idea is:

- Performance should run a fixed number of cases with fixed data sizes so that when the test is run on different platforms they can be compared
- Stress should run random cases with random sizes many, many times so that we can see if the system will fail

I have found a bunch of bugs in both the API and the OS that I have been working on thanks to the tests you have created so far. Thanks!

**Comment by shomagan** [comment]
Yes, I read the READMY file, in the previous tests it is possible to use rand() for the performance test, I will correct where necessary.How do I understand the most voluminous test should be stress test?

**Comment by tyler-gilbert** [comment]
Yes. The stress test should be the biggest. The performance test shouldn't
last more than a few seconds. That we many performance tests can be run and
compared in a short period of time.
